entries.py

```diff
@@ -18,7 +34,7 @@
 	pronunciation="", 
 	seealso="Biodata Catalyst")
 
-BDCat = GlossEntry("BDCat", 
+BDCat = GlossEntry("BD Catalyst", 
```

Comment:
I've heard BDCat several times, but I've almost never heard BD Catalyst. Which one is more common in your experience?

Reply:
BDCat is not an allowable abbreviation per the NHLBI style guide. Only BDC and BD Catalyst.

Reply:
Is the NHLBI style guide public? Might be a good idea to link it somewhere (probably not in this document, but maybe elsewhere, or in our internal docs).